README.md

```diff
@@ -893,6 +893,12 @@ Example situation: (2♥️)-X-?
 * 3♠️ Natural and game forcing
 * 3NT To play
 
+## Cue bid of an opposing weak 2
+
+After the opponents open with a weak 2, the direct cue bid is a stopper ask,
+showing a hand that is looking for 3NT but lacks a stopper and is unsuitable
```

Comment:
insert comma after "3NT" and before "but"

Reply:
this certainly implies a strong and likely balanced hand (else has a running suit); are we looking at 18+?